global.json

```diff
@@ -1,11 +1,11 @@
 {
   "sdk": {
-    "version": "9.0.100-preview.7.24407.12",
+    "version": "9.0.100-rc.1.24452.12",
     "rollForward": "major",
     "allowPrerelease": true
   },
   "tools": {
-    "dotnet": "9.0.100-preview.7.24407.12",
+    "dotnet": "9.0.100-rc.1.24452.12",
     "runtimes": {
       "dotnet/x64": [
```

Comment:
I feel like these might need some updates, @joperezr ?

Reply:
Yes, but we should do it in a separate PR.